sickbeard/subtitles.py

```diff
@@ -70,7 +84,8 @@ def sorted_service_list():
     for current_service in sickbeard.SUBTITLES_SERVICES_LIST:
         if current_service in provider_manager.names():
             new_list.append({'name': current_service,
-                             'url': PROVIDER_URLS[current_service] if current_service in PROVIDER_URLS else lmgtfy % current_service,
+                             'url': PROVIDER_URLS[current_service]
```

Comment:
This should really be like so to prevent long line length.

````
new_list.append({
    'name': current_service,
     'url': PROVIDER_URLS[current_service]
     if current_service in PROVIDER_URLS else lmgtfy % current_service,
     'image': current_service + '.png',
     'enabled': sickbeard.SUBTITLES_SERVICES_ENABLED[current_index] == 1
 })
````